extension/src/cli/index.ts

```diff
@@ -180,8 +196,8 @@ export class Cli extends Disposable implements ICli {
       {
         ...baseEvent,
         duration,
-        exitCode: cliError.exitCode,
-        stderr: cliError.stderr
+        errorOutput: all || cliError.stderr,
```

Comment:
[F] If we have captured a valid all string (length > 0) then we will use that, otherwise use the stderr from the generated error. I feel like this way there will always be something provided to the output channel